### ISSUE TITLE
wa2vec-u releasing the set of hyperparameters for LibriSpeech and MLS languages?

### ISSUE THREAD

**Issue by JeromeNi** [pytorch/fairseq#3679]
#### What is your question?
1. Is it possible to release the yaml files that contains the best hyperparameters for MLS languages (possibly as well as LibriSpeech), both for GAN training and Kaldi word-lm decoding? Running the model for 40 different hyperparameter setting basically takes way too long to replicate the results. The default one provided in the repo with a random seed of 0 seems too far from obtaining the reported results. For example, the Viterbi PER I got just after GAN training using the default hyperparameters provided in the repo's yaml file is 62% and way too high for the language model fusion to achieve near a 58% WER as reported in the paper.
2. What part of unlabeled 100h audio do you use from the MLS?
3. Will distributed training on more than 1 GPUs by changing world_size but at the same time reducing batch_size (so that the total batch size across multiple GPU is still 160) affect the result?
